caas/kubernetes/provider/k8swatcher.go

```diff
@@ -16,24 +17,26 @@ import (
 	"github.com/juju/juju/core/watcher"
 )
 
-// kubernetesWatcher reports changes to kubernetes
+// KubernetesWatcher reports changes to kubernetes
 // resources. A native kubernetes watcher is passed
 // in to generate change events from the kubernetes
 // model. These events are consolidated into a Juju
 // notification watcher event.
-type kubernetesWatcher struct {
+type KubernetesWatcher struct {
```

Comment:
instead of exporting this can we add a line to export_test.go

Reply:
now it's exposed in export_test